release.md

```diff
@@ -0,0 +1,19 @@
+# Scaffolding Release Process
+
+## Prerequisites
+
+You should be part of the `scaffolding-codeowners` or `sigstore-oncall` groups, which are defined within the
+[community repo](https://github.com/sigstore/community/blob/main/github-sync/github-data/users.yaml).
+
+## Steps
+
+1. Visit <https://github.com/sigstore/scaffolding/releases/new>
+1. Click on the `Choose a tag` drop-down
+1. For the tag name, use a new version number, for instance: v0.4.11
+1. Click on the `+ Create new tag: v0.4.11 on publish` button that appears.
```

Comment:
we should push a tag instead create a release from the UI, because we have a job that runs when we create a tag and at the end of the process it pushes the artifacts and creates a GitHub release, so creating a release in the beginning is not needed.

Reply:
So, just drop the lines L13-L??

Should push tag be done from the UI? As in creating a new tag, or use git-fu?
@cpanato

Reply:
I usually create/push the tag with git, which kicks off everything, and then once it's done turn the draft github release into a for realz release. I might have been doing it wrong this whole time though 😅